### DISCUSSION TITLE
Video aspect ratio changes

### DISCUSSION THREAD

**Discussion by realsmoky** [n00mkrad/flowframes#217] [Q&A]
Hello! I am pretty new to this piece of software, and not generally knowledgeable about video editing.

I downloaded a 720p YT video that seems to have a 16:9 aspect ratio (seeing as when I fullscreen it, I have no black bars). However, after running Flowframes, the new video has black bars and is squeezed.

How can I fix this?

Thank you!

**Comment by xavierhales29** [comment]
Hey got any solutions?
